docs/sources/flow/reference/components/prometheus.exporter.statsd.md

```diff
@@ -0,0 +1,108 @@
+---
+# NOTE(rfratto): the title below has zero-width spaces injected into it to
+# prevent it from overflowing the sidebar on the rendered site. Be careful when
+# modifying this section to retain the spaces.
+#
+# Ideally, in the future, we can fix the overflow issue with css rather than
+# injecting special characters.
+
+title: prometheus.exporter.statsd
+---
+
+# prometheus.exporter.statsd
+The `prometheus.exporter.statsd` component embeds
+[statsd_exporter](https://github.com/prometheus/statsd_exporter) for collecting StatsD-style metrics and exporting them as Prometheus metrics.
+
+## Usage
+
+```river
+prometheus.exporter.statsd "LABEL" {
+    
+}
+```
+
+## Arguments
+The following arguments can be used to configure the exporter's behavior.
+All arguments are optional. Omitted fields take their default values.
+
+Name | Type | Description | Default | Required
+---- | ---- | ----------- | ------- | --------
+`listen_udp`                                      | `string`       | The UDP address on which to receive statsd metric lines. Use "" to disable it. | `9125` | mo
+`listen_tcp`                                      | `string`       | The UDP address on which to receive statsd metric lines. Use "" to disable it. | `9125` | no
+`listen_unixgram`                                 | `string`       | The Unixgram socket path to receive statsd metric lines in datagram. Use "" to disable it. | | no
+`unix_socket_mode`                                | `string`       | The permission mode of the unix socket. | `755` | no
+`mapping_config_path`                             | `string`       | The path to a YAML mapping file used to translate specific dot-separated StatsD metrics into labeled Prometheus metrics. | | no
+`read_buffer`                                     | `int`          | Size (in bytes) of the operating system's transmit read buffer associated with the UDP or Unixgram connection. | | no
+`cache_size`                                      | `int`          | Maximum size of your metric mapping cache. Relies on least recently used replacement policy if max size is reached. | `1000` | no
+`cache_type`                                      | `string`       | Metric mapping cache type. Valid options are "lru" and "random". | `lru` | no
+`event_queue_size`                                | `int`          | Size of internal queue for processing events. | `10000` | no
+`event_flush_threshold`                           | `int`          | Number of events to hold in queue before flushing. | `1000`| no
+`event_flush_interval`                            | `string`       | Maximum time between event queue flushes. | `200ms`| no
+`parse_dogstatsd_tags`                            | `string`       | Parse DogStatsd style tags. Enabled by default. | `true`| no
```

Comment:
The default of `true` also indicates that it is enabled by default, and this makes it a bit shorter. (also applies to the next 3 lines)
```suggestion
`parse_dogstatsd_tags`                            | `string`       | Parse DogStatsd style tags. | `true`| no
```